views/participant.html.twig

```diff
@@ -208,7 +208,7 @@
                         {% set errorTitle = 'The Basics survey not complete' %}
                         {% set errorMessage = 'This participant has consented but has not completed "The Basics" survey in the participant app and is therefore ineligible for physical measurements and biospecimen collection. Please direct the participant to complete "The Basics" survey in the participant application to enable in person enrollment or contact the help desk if needed.' %}
                     {% elseif participant.statusReason == 'withdrawal' %}
-                        {% set errorTitle = 'Withdrawn participant' %}
+                        {% set errorTitle = 'Withdrawn participant: ' ~ participant.withdrawalTime|date('n/j/Y g:ia', app.userTimezone) %}
```

Comment:
I think we discussed that we should display the withdrawal date/time in the error box at the bottom of the message, similar to the withdrawal reason.

Something like:

```twig
<p>{{ errorMessage }}</p>
{% if participant.statusReason == 'withdrawal' %}
    <p><strong>Withdrawal time:</strong> {{ participant.withdrawalTime|date('n/j/Y g:ia', app.userTimezone) }}</p>
{% endif %}
{% if participant.statusReason == 'withdrawal' and participant.withdrawalReason is not empty %}
    <p>
        <strong>Reason:</strong> {{ participant.withdrawalReason }}
        {% if participant.withdrawalReasonJustification is not empty %}
            <br /><strong>Justification:</strong> {{ participant.withdrawalReasonJustification }}
        {% endif %}
    </p>
{% endif %}